TypeScript/channelType.ts

```diff
@@ -0,0 +1,43 @@
+//@ts-nocheck
+/**
+ * This plugin checks if a channel is the specified type (does not accept)
+ *
+ * @author @NeoYaBoi [<@762918086349029386>]
+ * @version 1.0.0
+ * @example
+ * ```ts
+ * import { channelType } from "../plugins/channelType";
+ * import { channelType } from "discord.js"
+ * import { commandModule } from "@sern/handler";
+ * export default commandModule({
+ *  plugins: [ channelType(channelType.GuildText, 'This cannot be used here') ],
+ *  execute: (ctx) => {
+ * 		//your code here
+ *  }
+ * })
+ * ```
+ */
+import { ChannelType } from "discord.js";
+import { CommandType, EventPlugin, PluginType } from "@sern/handler";
+export function channelType(
+  channelType: ChannelType,
+  onFail: string

```

Comment:
make this onFail optional, if nothing is provided it will be silently stopped from running command